fastlane/Fastfile

```diff
@@ -128,6 +134,7 @@ end
 
 private_lane :sources_matrix do
   {
+    unit: ['stream-chat-android-client/', 'stream-chat-android-client-test/', 'stream-chat-android-compose/', 'stream-chat-android-core/', 'stream-chat-android-markdown-transformer/', 'stream-chat-android-offline/', 'stream-chat-android-state/', 'stream-chat-android-ui-common/', 'stream-chat-android-ui-components/', 'stream-chat-android-ui-utils/'],
```

Comment:
What does this list represent?

Reply:
This list represents an impact analysis. If any of this paths were touched in the PR, then the check will be executed on CI. E.g.: if you updated `README.md` that sonar check won't be executed. But if you updated some file under `stream-chat-android-client/` the check will be executed.

The `unit` key reflects unit tests, even though we do not use impact analysis for unit tests at the moment, I thought it still makes sense to use it for sonar analysis.

Reply:
Do we need to include `stream-chat-android-client-test` on this list??
This module is only used to provide "mother helpers" used on the rest of the tests modules, but it is not code perse.

Reply:
I think you are right. Let me create a separate list for the sonar source matrix

Reply:
Done ✅